CHANGELOG.md

```diff
@@ -30,6 +31,7 @@ Release report: https://github.com/wazuh/wazuh/issues/17198
 
 Wazuh commit: https://github.com/wazuh/wazuh/commit/b2901d5086e7a073d89f4f72827e070ce3abd8e8
 Release report: https://github.com/wazuh/wazuh/issues/17004
+>>>>>>> 4.4
```

Comment:
```suggestion
```

Reply:
Done in [8aa43a5](https://github.com/wazuh/wazuh-qa/pull/4205/commits/8aa43a5c4a87fade9a31db57262fc41304a733e2)